### ISSUE TITLE
Infinite Looping items?

### ISSUE THREAD

**Issue by moeindev** [3ph/CollectionPickerView#7]
Is it possible to have infinite looping items? I've tried combining with [InfiniteLayout](https://github.com/arnauddorgans/InfiniteLayout) but I failed :(

also check my [issue](https://github.com/arnauddorgans/InfiniteLayout/issues/14)

**Comment by 3ph** [comment]
You mean that you would see first item following the last item? I can have a look if it's possible to implement.

**Comment by moeindev** [comment]
yes exaclty! and also see the last item above the first item, like a loop view
